dist/profile/templates/buildmaster/casc/clouds.yaml.erb

```diff
@@ -14,7 +14,7 @@ jenkins:
       vmTemplates:
       - agentLaunchMethod: "SSH"
         agentWorkspace: "<%= @agents_setup[agent["os"]]["agentDir"] %>"
-        credentialsId: "jenkinsvmagents-userpass"
+        credentialsId: "<%= agent["credentialsId"] %>"
```

Comment:
questionable if this needs to be configurable

Reply:
the credential is different between ci.j, trusted.ci and others (both its id and the value), hence the need to configure it.

Reply:
add new credential / rename existing ones, =/. easier in general if they're the same across environments.